UndertaleModTool/Scripts/Community Scripts/TouchControls_data/gml_Object_obj_mobilecontrols_Create_0.gml

```diff
@@ -1,13 +1,18 @@
-if (string_lower(game_display_name) == "undertale")
+if (font_exists(fnt_main))
```

Comment:
I think this would cause a variable not defined error, because compiling this code on a game without fnt_main would turn it into an undefined instance variable named fnt_main. To work around this, you might just need to edit this code in the csx script (after importing it) based on whether the font exists.